pvlib/tests/test_modelchain.py

```diff
@@ -447,22 +447,30 @@ def acdc(mc):
     mc.ac = mc.dc
 
 
+# TODO in v0.9: remove 'snlinverter', 'adrinverter' since these values for
+# ModelChain.ac_model are superseded by 'sandia' and 'adr'
 @pytest.mark.parametrize('ac_model', [
-    'snlinverter', pytest.param('adrinverter', marks=requires_scipy),
+    'sandia', 'snlinverter', pytest.param('adrinverter', marks=requires_scipy),
+    pytest.param('adr', marks=requires_scipy),
     'pvwatts'])
 def test_ac_models(sapm_dc_snl_ac_system, cec_dc_adr_ac_system,
                    pvwatts_dc_pvwatts_ac_system, location, ac_model,
                    weather, mocker):
     ac_systems = {'snlinverter': sapm_dc_snl_ac_system,
+                  'sandia': sapm_dc_snl_ac_system,
                   'adrinverter': cec_dc_adr_ac_system,
+                  'adr': cec_dc_adr_ac_system,
                   'pvwatts': pvwatts_dc_pvwatts_ac_system}
+    ac_method_name = {'snlinverter': 'snlinverter',
+                      'sandia': 'snlinverter',
+                      'adrinverter': 'adrinverter',
+                      'adr': 'adrinverter',
+                      'pvwatts': 'pvwatts_ac'}
     system = ac_systems[ac_model]
 
```

Comment:
Ideally we would test that the deprecation warning is raised if the ac method name is snlinverter or adrinverter. Ok with me to skip it though since the code looks correct and is not too complicated.

Reply:
I'll add a test for the deprecation warning if ModelChain.ac_model is assigned 'snlinverter' or 'adrinverter'

The ModelChain and PVSystem methods are still snlinverter and adrinverter.  Do we want to rename those as well? I favor doing that, but in a separate PR.

Reply:
I agree we should rename the methods in a separate PR.